examples/native/components/ActionExample/Actions.stories.tsx

```diff
@@ -22,3 +23,37 @@ export const AnotherAction: ActionButtonStory = (args) => <ActionButton {...args
 AnotherAction.argTypes = {
   onPress: { action: 'pressed a different button' },
 };
+
+export const ListBasic: ActionButtonStory = (args) => (
```

Comment:
I think it would be better to create a separate story instead of using the story intended for testing actions